packages/compiler/index.ts

```diff
@@ -52,6 +52,7 @@ export function compileVineTypeScriptFile(
     ],
   })
   const vineFileCtx: VineFileCtx = {
+    originCode: code,
```

Comment:
有什么可以帮你的吗

中译英：

有点不太建议这样做，可以直接用 那个 MagicString 的 .original。
因为这样可能算同一个字符串保存两份，（虽然性能考量问题不大）但是不太好

I don't recommend doing it that way. You can directly use the .original property of the MagicString. Although it may result in storing the same string twice (although performance-wise it's not a big concern), it's not ideal.